src/lib/list/list.ts

```diff
@@ -146,9 +158,9 @@ export class MdListItem extends _MdListItemMixinBase implements AfterContentInit
   constructor(private _renderer: Renderer2,
               private _element: ElementRef,
               @Optional() private _list: MdList,
-              @Optional() navList: MdNavListCssMatStyler) {
```

Comment:
Are we still using `MdNavListCssMatStyler` for anything or can we remove it now?